cairo/ethereum_rlp/rlp.cairo

```diff
@@ -789,14 +791,20 @@ func decode_to_bytes{range_check_ptr}(encoded_bytes: Bytes) -> Bytes {
     let cond = is_le(encoded_bytes.value.data[0], 0xB7);
     if (cond != 0) {
         let len_raw_data = encoded_bytes.value.data[0] - 0x80;
-        assert [range_check_ptr] = len_raw_data;
-        let range_check_ptr = range_check_ptr + 1;
-        assert [range_check_ptr] = encoded_bytes.value.len - len_raw_data;
-        let range_check_ptr = range_check_ptr + 1;
+        with_attr error_message("DecodingError") {
+            assert [range_check_ptr] = len_raw_data;
+            let range_check_ptr = range_check_ptr + 1;
+        }
+        with_attr error_message("DecodingError") {
+            assert [range_check_ptr] = encoded_bytes.value.len - (len_raw_data + 1);
+            let range_check_ptr = range_check_ptr + 1;
+        }
```

Comment:
```suggestion
        with_attr error_message("DecodingError") {
            assert [range_check_ptr] = len_raw_data;
            let range_check_ptr = range_check_ptr + 1;
            assert [range_check_ptr] = encoded_bytes.value.len - (len_raw_data + 1);
            let range_check_ptr = range_check_ptr + 1;
        }
```

Reply:
I prefer to have it similar to python where each assertion can raise with a specific error - easier to read